ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js

```diff
@@ -84,6 +84,12 @@
                 widget.addEventListener('change', (e) => {
                     this._model.value = e.target.value;
                 });
+                widget.addEventListener('focus', (e) => {
+                    this.setActive();
+                });
+                widget.addEventListener('blur', (e) => {
+                    this.setInactive();
```

Comment:
Can you update all the component read me with these data attributes ?

Reply:
Hi Rishi its documented in the js docs 
https://opensource.adobe.com/aem-core-forms-components/module-FormView-Constants.html#:~:text=string-,%3Cstatic%3E%20DATA_ATTRIBUTE_ENABLED%20%3Astring,-Data%20attribute%20to

I don't find an appropriate place in readme of components to add this.

Reply:
You need to add this in all the component readme.md file in the data attributes section

Reply:
Updated!